### ISSUE TITLE
0.5092167652

### ISSUE THREAD

**Issue by oneonlee** [oneonlee/NLP-author-classification#3]
- batch_size = 16
- epochs = 3
- learning_rate = 5e-5
- **scheduler = torch.optim.lr_scheduler.StepLR(optimizer, step_size=1, gamma=0.1)**

learning_rate를 고정값인 `1e-5`(#2) 대신 StepLR 스케줄러를 사용했을 때, 성능(LogLoss)이 `0.589376257`(#2)에서 `0.5092167652`로 증가함

**Comment by oneonlee** [comment]
epoch을 `3`에서 `6`(#4)으로 사용했을 때, 성능(LogLoss)이 `0.5092167652`에서 `0.4508949222`(#4)로 증가함
